README.md

```diff
@@ -71,36 +71,95 @@ a string in place of an object for the `start`, `stop`, and `refresh` methods.
 This string will be assumed to be the `exec` property of that method.
 
 
-# Example
+## Example
 
-    $ cat bapi.json 
-    {
-        "ident": "bapi",
-        "label": "Boilerplate API",
-        "start": "node bapi.js"
-    }
+    $ cat bapi.json
+
+``` json
+{
+    "ident": "bapi",
+    "label": "Boilerplate API",
+    "start": "node bapi.js"
+}
+```
+
+    $ ./smfgen < bapi.json
+
+``` xml
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!-- 
+    Manifest automatically generated by smfgen.
+ -->
+<service_bundle type="manifest" name="application-bapi" >
+    <service name="application/bapi" type="service" version="1" >
+        <create_default_instance enabled="true" />
+        <dependency name="dep1" grouping="require_all" restart_on="error" type="service" >
+            <service_fmri value="svc:/milestone/multi-user:default" />
+        </dependency>
+        <exec_method type="method" name="start" exec="node bapi.js &amp;" timeout_seconds="10" />
+        <exec_method type="method" name="stop" exec=":kill" timeout_seconds="30" />
+        <template >
+            <common_name >
+                <loctext xml:lang="C" >Boilerplate API</loctext>
+            </common_name>
+        </template>
+    </service>
+</service_bundle>
+```
 
-    $ ./smfgen < bapi.json 
-    <?xml version="1.0"?>
-    <!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
-    <!-- 
-        Manifest automatically generated by smfgen.
-     -->
-    <service_bundle type="manifest" name="application-bapi" >
-        <service name="application/bapi" type="service" version="1" >
-            <create_default_instance enabled="true" />
-            <dependency name="dep1" grouping="require_all" restart_on="error" type="service" >
-                <service_fmri value="svc:/milestone/multi-user:default" />
-            </dependency>
-            <exec_method type="method" name="start" exec="node bapi.js &amp;" timeout_seconds="10" />
-            <exec_method type="method" name="stop" exec=":kill" timeout_seconds="30" />
-            <template >
-                <common_name >
-                    <loctext xml:lang="C" >Boilerplate API</loctext>
-                </common_name>
-            </template>
-        </service>
-    </service_bundle>
+# CLI Input
+
+The following options can be specified over the CLI:
+
+-  `-d`, `--cwd` becomes `data.start.working_directory`
+-  `-e`, `--env` appends to `data.start.environment`
```

Comment:
nit, should we mention that these also apply to the stop method?
(and refresh?)

Reply:
Also, given e.g. data.exec and data.timeout works, shouldn't these say that? instead of mentioning start